src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss

```diff
@@ -31,4 +31,26 @@
   overflow-x: scroll;
   width: 100%;
   display: grid;
-}
\ No newline at end of file
+}
+
+$base-color: #00BAEE;
+$hover-color: darken($base-color, 20);
```

Comment:
If these variables are only valid and useful for the .is-url class, you may want to rename them to something like `url-base-color` & `url-hover-color`.